INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:27:48] "GET / HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:27:48] "GET /static/css/main.css HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:27:48] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:app:<int:post_id> request successfull
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:27:54] "GET /1 HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:27:54] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:28:07] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:28:47] "[33mGET /test HTTP/1.1[0m" 404 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:40:51] "GET / HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:40:51] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /1 [GET]
Traceback (most recent call last):
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 73, in post
    app.logger.info({{post['title']}})
TypeError: unhashable type: 'set'
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:40:54] "[35m[1mGET /1 HTTP/1.1[0m" 500 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:app:<function <lambda> at 0x7f3b21dafd30>
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:49:17] "GET /1 HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:49:17] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:app:<function <lambda> at 0x7f3b21dafd30>
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:49:31] "GET /9 HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:49:31] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:51:26] "GET / HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:51:26] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:app:"{title}" request successful
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:51:28] "GET /1 HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 12:51:28] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:app:Exception on /3 [GET]
Traceback (most recent call last):
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 73, in post
    app.logger.info( post + 'request successful')
TypeError: unsupported operand type(s) for +: 'sqlite3.Row' and 'str'
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 13:01:12] "[35m[1mGET /3 HTTP/1.1[0m" 500 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:app:Exception on /3 [GET]
Traceback (most recent call last):
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 73, in post
    app.logger.info( print(post) + 'request successful')
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 13:02:01] "[35m[1mGET /3 HTTP/1.1[0m" 500 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:app:<sqlite3.Row object at 0x7ff541300050>request successful
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 13:02:37] "GET /2 HTTP/1.1" 200 -
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 13:02:37] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:app:Exception on /9 [GET]
Traceback (most recent call last):
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/andrew/.local/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 73, in post
    app.logger.info( str(post.title) + 'request successful')
AttributeError: 'sqlite3.Row' object has no attribute 'title'
INFO:werkzeug:172.31.48.1 - - [17/Oct/2022 13:03:47] "[35m[1mGET /9 HTTP/1.1[0m" 500 -
10/17/2022 01:23:05 PM [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:3111
 * Running on http://172.31.62.19:3111
10/17/2022 01:23:05 PM [33mPress CTRL+C to quit[0m
10/17/2022 01:23:07 PM Metrics request successfull
10/17/2022 01:23:07 PM 172.31.48.1 - - [17/Oct/2022 13:23:07] "GET /metrics HTTP/1.1" 200 -
